kubernetes/gke-utility/argocd/clusters.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
     argocd.argoproj.io/secret-type: cluster
     clusterType: prow
     environment: prod
+    prow-namespace: default
```

Comment:
Very tiny nit: maybe go with `prowNamespace` to match other variables, mainly `clusterType`

Reply:
Fixed